packages/protocol/test/common/accounts.ts

```diff
@@ -8,6 +6,8 @@ import {
   assertRevert,
   NULL_ADDRESS,
 } from '@celo/protocol/lib/test-utils'
+import { parseSolidityStringArray } from '@celo/utils/lib/parsing'
+import { upperFirst } from 'lodash'
```

Comment:
you don't seem to be using these imports

Reply:
the imports are being used  e.g. see line 466 , line 313